FreeRTOS/FreeRTOS-Kernel
# Update example cmake project path

Description
-----------
The PR updates the path for the cmake project and and sample configuration.

Test Steps
-----------
Tested by building the demo on 64 bit Linux using GCC:

```
(base) [δ:0s T:13:47:11]ubuntu@ip-172-31-69-236:~/Projects/FreeRTOS-Kernel$ cd build
(base) [δ:0s T:13:47:37]ubuntu@ip-172-31-69-236:~/Projects/FreeRTOS-Kernel/build$ rm -rf *
(base) [δ:0s T:13:47:41]ubuntu@ip-172-31-69-236:~/Projects/FreeRTOS-Kernel/build$ cmake ../examples/cmake_example/
-- The C compiler identification is GNU 9.4.0
-- The CXX compiler identification is GNU 9.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ubuntu/Projects/FreeRTOS-Kernel/build
(base) [δ:1s T:13:47:55]ubuntu@ip-172-31-69-236:~/Projects/FreeRTOS-Kernel/build$ make
Scanning dependencies of target freertos_kernel_port
[  7%] Building C object FreeRTOS-Kernel/portable/CMakeFiles/freertos_kernel_port.dir/template/port.c.o
[ 15%] Linking C static library libfreertos_kernel_port.a
[ 15%] Built target freertos_kernel_port
Scanning dependencies of target freertos_kernel
[ 23%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/croutine.c.o
[ 30%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/event_groups.c.o
[ 38%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/list.c.o
[ 46%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/queue.c.o
[ 53%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/stream_buffer.c.o
[ 61%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/tasks.c.o
[ 69%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/timers.c.o
[ 76%] Building C object FreeRTOS-Kernel/CMakeFiles/freertos_kernel.dir/portable/MemMang/heap_4.c.o
[ 84%] Linking C static library libfreertos_kernel.a
[ 84%] Built target freertos_kernel
Scanning dependencies of target example
[ 92%] Building C object CMakeFiles/example.dir/main.c.o
[100%] Linking C executable example
[100%] Built target example
(base) [δ:1s T:13:47:58]ubuntu@ip-172-31-69-236:~/Projects/FreeRTOS-Kernel/build$ 
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
